controllers/front/AddressesController.php

```diff
@@ -59,11 +59,11 @@ public function initContent()
         $this->setTemplate('customer/addresses.tpl');
     }
 
-    public function getBreadcrumb()
+    public function getBreadcrumbLinks()
```

Comment:
it's public api, any chance that a module use it ? Can we keep and deprecate ``getBreadcrumb`` else ?